apps/web/playwright/teams.e2e.ts

```diff
@@ -12,30 +13,27 @@ test.describe("Teams", () => {
   test("Can create teams via Wizard", async ({ page, users }) => {
     const user = await users.create();
     const inviteeEmail = `${user.username}+invitee@example.com`;
+    let slug: string | RegExp;
     await user.apiLogin();
     await page.goto("/teams");
 
     await test.step("Can create team", async () => {
-      // Click text=Create Team
       await page.locator("text=Create Team").click();
       await page.waitForURL("/settings/teams/new");
-      // Fill input[name="name"]
       await page.locator('input[name="name"]').fill(`${user.username}'s Team`);
-      // Click text=Continue
+      slug = await page.locator(`[name='slug']`).innerText();
       await page.locator("text=Continue").click();
       await page.waitForURL(/\/settings\/teams\/(\d+)\/onboard-members$/i);
       await page.waitForSelector('[data-testid="pending-member-list"]');
       expect(await page.locator('[data-testid="pending-member-item"]').count()).toBe(1);
     });
 
     await test.step("Can add members", async () => {
-      // Click [data-testid="new-member-button"]
       await page.locator('[data-testid="new-member-button"]').click();
-      // Fill [placeholder="email\@example\.com"]
       await page.locator('[placeholder="email\\@example\\.com"]').fill(inviteeEmail);
-      // Click [data-testid="invite-new-member-button"]
       await page.locator('[data-testid="invite-new-member-button"]').click();
       await expect(page.locator(`li:has-text("${inviteeEmail}")`)).toBeVisible();
+      await assertToasterText(page, `toast-success`, `${inviteeEmail} has been invited`);
```

Comment:
To remain consistent with the rest of tests.

```suggestion
      const toast = await page.waitForSelector('[data-testid="toast-success"]');
      expect(toast).toBeTruthy();
```